src/mwparserfromhell/nodes/template.py

```diff
@@ -330,6 +326,23 @@ def add(
             self.params.append(param)
         return param
 
+    def update(self, params):
+        """Update the template with multiple parameters at once.
+        
+        This method accepts a list of parameter dictionaries, each containing
+        the same arguments as :meth:`add`. It will process each parameter
+        in order using the same logic as :meth:`add`.
+        """
+        for param in params:
+            self.add(
```

Comment:
This can be done just like this:

```suggestion
            self.add(**param)
```